### PR TITLE
snippets lambda

### PR DESCRIPTION
**Pull request by metadave**
This PR renames the `cdn` dir to `prod_cdn` and `stage_cdn` so that Lambda@Edge functions aren't deployed to prod, as it's now [in use](https://github.com/mozmeao/infra/issues/1042#issuecomment-528045185). Once we're ready to deploy to prod, the `stage_cdn` dir can be removed, and `prod_cdn` can be renamed to `cdn`.

@glogiotatidis I used one of the example Python lambda functions from [here](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-examples.html), but it's returning a 503 for some reason. I'll try to troubleshoot a bit more later in the week.

```
curl -I https://d177vs5g6znz6f.cloudfront.net/foobar123
HTTP/1.1 503 Service Unavailable
```

### CODE REVIEW

File: apps/snippets/infra/global/lambda.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+def lambda_handler(event, context):
+     response = event['Records'][0]['cf']['response']
```
Comment (glogiotatidis) [inline apps/snippets/infra/global/lambda.py]: This is a Origin Response example, that's why it wasn't working. I've made some changes via the console testing out some things. Once I get comfortable with lambda@edge i'll update terraform

Comment (metadave) [reply apps/snippets/infra/global/lambda.py]: that's what I get for copy/pasting code :-) thanks for looking into this @glogiotatidis!

**Comment by glogiotatidis** [comment]
Can I have a couple of bullets on how to tf apply this?

**Comment by metadave** [comment]
@glogiotatidis sent you a link to a gdrive dir that has a doc + video on how to get things running.

**Comment by glogiotatidis** [comment]
@metadave Updated this branch with the current lambda setup and with a commit to fix #1167 . This is applied on production

**Comment by glogiotatidis** [comment]
Also again thanks for the toolkit, really helpful!

**Pull request merged**